gulp/tasks/buildCustomJs.js

```diff
@@ -35,7 +36,21 @@ gulp.task('custom-js', ['select-view', 'custom-html-templates'],() => {
 });
 
 function buildByConcatination() {
-    return gulp.src([buildParams.customModulePath(),buildParams.mainPath(),buildParams.customNpmJsPath(),buildParams.customNpmDistPath(),'!'+buildParams.customPath(),'!'+buildParams.customNpmJsModulePath(),'!'+buildParams.customNpmJsCustomPath()])
+    return gulp.src([
+        buildParams.customModulePath(),
+        buildParams.mainPath(),
+        buildParams.customNpmJsPath(),
+        buildParams.customNpmDistPath(),
+        '!'+buildParams.customPath(),
+        '!'+buildParams.customNpmJsModulePath(),
+        '!'+buildParams.customNpmJsCustomPath()
+    ])
+        .pipe(order([
+            buildParams.mainJsPath(),
+            buildParams.customModulePath(),
```

Comment:
Hi,

Why did you put `buildParams.customModulePath()` after `buildParams.mainJsPath()` in ?

Reply:
Hi Gilax, 

I just checked and the custom.module.js file can also appear after the main.js file (At least in our case). 

However, if the custom.module.js file contains some dependencies or references to variables defined in the main.js file then it should appear after the main.js file in the concatenation order. 

I think that was my rational.